src/main/java/net/geforcemods/securitycraft/datagen/RecipeGenerator.java

```diff
@@ -331,6 +331,13 @@ protected final void registerRecipes(Consumer<IFinishedRecipe> consumer)
 		.key('G', Tags.Items.FENCE_GATES_WOODEN)
 		.addCriterion("has_iron", hasItem(Tags.Items.INGOTS_IRON))
 		.build(consumer);
+		ShapedRecipeBuilder.shapedRecipe(SCContent.REINFORCED_LEVER.get())
+		.patternLine("S")
+		.patternLine("C")
+		.key('S', Tags.Items.RODS_WOODEN)
+		.key('C', SCContent.REINFORCED_COBBLESTONE.get())

```

Comment:
This should use a tag containing reinforced cobblestone and reinforced moss stone, just like the vanilla lever.

Reply:
You forgot to rerun the `runData` task again.